File: LeetCode/Contains Duplicate/ContainsDuplicate.cpp
#include<bits/stdc++.h>

using namespace std;

bool containsDuplicate(vector<int> nums){
	return nums.size() > set<int>(nums.begin(), nums.end()).size();
}

int main(){
	int n;
	cin >> n;
	vector<int> v[n];
	for(int i=0; i<n; i++)
		cin >> v[n];

	bool result = containsDuplicate(v);
	cout << result << endl;
	return 0;
}

File: LeetCode/Contains Duplicate/ContainsDuplicate.js
var containsDuplicate = function(nums) {
    let set = new Set();
    
    for (let i = 0; i < nums.length; i++) {
        if (set.has(nums[i])) {
            return true;
        }
        set.add(nums[i]);
    }
    return false;
};

let readline = require('readline');
let read = readline.createInterface({
	input: process.stdin,
	output: process.stdout
});

read.question('', function(array){
	array = array.split(' ').map((value) => {
		return Number(value);
	});
	let result = containsDuplicate(array);
	console.log(result);
	read.close();
});